serverless.yml

```diff
@@ -206,6 +206,9 @@ custom:
     external: ['knex', 'pg', 'sharp']
     bundle: true
     minify: ${self:custom.minifyBuild.${self:provider.stage}}
+    # exclude: ['./node_modules/*', '*.test.js']
+    watch:
+      ignore: ['.esbuild', 'dist', 'node_modules', '.build', 'static', "*.test.js"]
```

Comment:
This is essentially the solution for this ticket. For me its helped the speed of running tests while `esbuild` is running 

See https://bugs.earthdata.nasa.gov/browse/EDSC-4236 I got tired of brining this along to all my tickets

Reply:
Default is `['.esbuild', 'dist', 'node_modules', '.build']` we'll want to include those in our override

Reply:
remove double quotes